client/src/app/formulaire/formulaire.component.spec.ts

```diff
@@ -75,15 +88,22 @@ describe('FormuleComponent', () => {
          axesAmeliorations.setValue("les axes ameliorations:.....")
           expect(axesAmeliorations.valid).toBeTruthy();
         }))
+
    it('Formulaire doit être valide quand tous les champs sont valides',waitForAsync(()=>{
     let nom = component.feedbackForm.controls['nom'];
     nom.setValue("Pompidor Pierre")
+
     let email = component.feedbackForm.controls['email'];
-    email.setValue("bnyat.azizsharif@zenika.com")
+    email.setValue("pierre.pompidor@exemple.com")
+
+    let receverEmail = component.feedbackForm.controls['receverEmail'];
+    receverEmail.setValue("marie.mettrand@exemple.com")
+
```

Comment:
```suggestion
    email.setValue("pierre.pompidor@example.com")

    let receverEmail = component.feedbackForm.controls['receverEmail'];
    receverEmail.setValue("marie.mettrand@example.com")

```
En anglais c'est avec un a le example

Reply:
Mais quand il s'agit d'expliquer on le fait pas en français? mais t'as raison en fait c'est un domain exemple pas l'explication.